private/tst/avb_streamhandler/CMakeLists.txt

```diff
@@ -33,9 +33,10 @@ add_executable( test_IasTestAvbStreamhandler
                 private/tst/avb_streamhandler/src/IasTestAvbConfigurationBase.cpp
                 private/tst/avb_streamhandler/src/IasTestAvbMain.cpp
                 private/tst/avb_streamhandler/src/IasTestAvbClockDriver.cpp
+                private/tst/avb_streamhandler/src/IasTestSystemdWatchdog.cpp
                 )
 
-target_compile_options( test_IasTestAvbStreamhandler PRIVATE -Wno-error )
+target_compile_options( test_IasTestAvbStreamhandler PRIVATE -Wno-error -Wsign-conversion)
```

Comment:
Please, add an issue so we can track the task of adding this to avb-sh build itself.